react-app/src/page.js

```diff
@@ -0,0 +1,63 @@
+import theme from "./theme";
+
+// styling for each page
+const page = {
+    body: {
+        fontFamily: "Montserrat",
+        margin: 16,
+        display: "flex",
+        flexDirection: "row",
+    },
+
+    content: {
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-between",
+        margin: 92,
+        width: "73.545%",
```

Comment:
I know it's not your code, but we probably need to change these CSS values in the future to make the website responsive. I'll bring this up in an issue.